### PR TITLE
Use uint16 for `EncodeDecimal`

### PR DESCRIPTION
**Pull request by Tang8330**
As per title.

### CODE REVIEW

File: lib/debezium/decimal_test.go [hunk 1]
```diff
@@ -86,7 +86,7 @@ func TestEncodeDecimal(t *testing.T) {
 	for _, testCase := range testCases {
 		encodedValue, err := EncodeDecimal(testCase.value, testCase.scale)
 		if testCase.expectedErr == "" {
-			decodedValue := DecodeDecimal(encodedValue, nil, testCase.scale)
+			decodedValue := DecodeDecimal(encodedValue, nil, int(testCase.scale))
```
Comment (nathan-artie) [inline lib/debezium/decimal_test.go:89]: It'd be nice to make make `DecodeDecimal` and `Decimal` use `int16` too.

**Review by nathan-artie — APPROVED** [review]

File: lib/debezium/decimal_test.go (hunk 1 above)
Comment (Tang8330) [reply lib/debezium/decimal_test.go:89]: Yeah I tried, it's pretty hard

Comment (nathan-artie) [reply lib/debezium/decimal_test.go:89]: What about int64 then?

**Comment by Tang8330** [comment]
The precision isn't the problem, it's the fact that we have lots of code
references that will need to specify

**Pull request merged**